### PR TITLE
Deliveroo Engineer Careers page

### PR DESCRIPTION
**Pull request by jphastings**
Add a careers page that pulls data in from Workable.

deliveroo.engineering/careers

### CODE REVIEW

**Comment by mezis** [comment]
Could we maybe fetch the jQuery blob from a CDN instead of hosting it?

**Comment by mezis** [comment]
Should we maybe restrict the list to product team roles? That'll help, as Workable's API is... slow.

File: careers/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+---
+<h1>Careers with Deliveroo</h1>
+<p class="tagline">Only awesome code.</p>
+
+<section id="careers" class="listings-loading">
+  <p>The Product Team naturally have a tremendous impact on the whole show at Deliveroo. So with that in mind, it’s only right that we hire the best of the bunch. That’s where you come in.</p>
```
Comment (stefandouganhyde) [inline careers/index.html]: Nitpick: `... naturally have a...` -> `... naturally has a...`?

**Comment by stefandouganhyde** [comment]
This is awesome. 😀

File: careers/index.html (hunk 1 above)
Comment (jphastings) [inline careers/index.html]: Good nit picking!

**Comment by jphastings** [comment]
@mezis sadly it doesn't look as though we can just pull in the stuff we want - their embeddable JS is fairly limited (plus I don't have access to admin Workable to get the full details, as there's private information in there only the HR department should have access to).

File: authors/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 layout: default
 ---
 <h1>Authors</h1>
-<p class="tagline">The members of the engineering team at Deliveroo who’ve contributed to the engineering blog.</p>
+<p class="tagline">Members of Deliveroo's engineering team who’ve contributed to the site.</p>
```
Comment (gregbeech) [inline authors/index.html]: We should probably add a bunch more people here, as a load of people have reviewed and written guidelines.

Comment (kapowaz) [inline authors/index.html]: Agreed! I think a better idea would be for the authors page to change to an Engineering Team page listing everyone in the team. The [Design blog](http://www.deliveroo.design) has some lovely team photos, taken with proper camera. I’d much prefer that to us having personally-sourced photos of varying quality.

File: _sass/local/_careers.scss [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        box-shadow: 0 2px 0 rgba(46, 51, 51, 0.1);
+        text-decoration: none;
+      }
+    }
+  }
+}
+
+#careers {
+  max-width: 800px;
+}
```
Comment (kapowaz) [inline _sass/local/_careers.scss]: Best to end with a newline here?

File: _sass/local/_careers.scss [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+      box-shadow: 0 2.5em 0 -1.3em;
+    }
+    40% {
+      box-shadow: 0 2.5em 0 0;
+    }
+  }
+}
+
+ul.jobs-list {
+  list-style-type: none;
```
Comment (kapowaz) [inline _sass/local/_careers.scss]: There’s a mixin `@include unstyled-list;` that’s good for doing this kind of thing.

File: _sass/local/_careers.scss [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+  padding: 0;
+
+  & > li {
+    padding: 1rem 0;
+    border-bottom: 1px solid color(soba, 40%);
+
+    &:after {
+      content: "";
```
Comment (kapowaz) [inline _sass/local/_careers.scss]: I’ll have to do some investigation into browser support for this kind of clearing fix, but I’ve always just used `overflow: auto;` on the containing element as a clearfix more recently — far less code to worry about. Is there some particular browser this is intended to help out with?

File: _sass/local/_careers.scss [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+      display: table;
+      clear: both;
+    }
+  }
+
+  h3 {
+    display: block;
+    float: left;
+    margin: 0;
+    font-family: 'Adelle Sans', "Helvetica Neue", Helvetica, Arial, sans-serif;
```
Comment (kapowaz) [inline _sass/local/_careers.scss]: There’s a variable with this specific font stack defined in _typography.scss: `$default-font`.

File: _sass/local/_careers.scss [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+    font-size: 18px;
```
Comment (kapowaz) [inline _sass/local/_careers.scss]: There’s a bunch of brand guidelines-defined named sizes in _typography.scss, so `font-size: font-size(extra-small-heading);` would be cool here.

File: _sass/local/_careers.scss [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+    line-height: 36px;
+    font-weight: 300;
+
+    a::after {
+      margin: 0;
```
Comment (kapowaz) [inline _sass/local/_careers.scss]: These properties will be ignored and overridden by the external link SCSS — are they still needed?

File: _sass/local/_careers.scss [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+    }
+
+    li.whr-apply {
+      float: right;
+      a {
+        color: white;
+        background-color: #440063;
+        border-radius: 2px;
+        padding: 10px 10px 8px;
+        box-shadow: 0 2px 0 rgba(46, 51, 51, 0.1);
```
Comment (kapowaz) [inline _sass/local/_careers.scss]: `box-shadow: 0 2px 0 color(mackerel, $transparency: 0.1);` maybe?

**Comment by kapowaz** [comment]
A few minor style points but otherwise LGTM! 👍

**Comment by kapowaz** [comment]
I’ve refactored the code based on my feedback, and also added some smaller breakpoints to the new careers page layout to help it work better at smaller screen sizes:

**320px**

img

**480px**

img

**640px**

img

**Pull request merged**